app/submitter/convert_payload_0_0_1.py

```diff
@@ -73,48 +82,55 @@ def convert_answers_to_payload_0_0_1(
     return data
 
 
-def _format_downstream_answer(answer_type, answer_value, answer_data):
-    if answer_type == "Date":
-        return (
-            datetime.strptime(answer_value, "%Y-%m-%d")
-            .replace(tzinfo=timezone.utc)
-            .strftime("%d/%m/%Y")
-        )
+def _format_downstream_answer(
+    answer_type: str, answer_value: AnswerValueTypes, answer_data: str
+) -> str:
+    if isinstance(answer_value, str):
+        if answer_type == "Date":
+            return (
+                datetime.strptime(answer_value, "%Y-%m-%d")
+                .replace(tzinfo=timezone.utc)
+                .strftime("%d/%m/%Y")
+            )
 
-    if answer_type == "MonthYearDate":
-        return (
-            datetime.strptime(answer_value, "%Y-%m")
-            .replace(tzinfo=timezone.utc)
-            .strftime("%m/%Y")
-        )
+        if answer_type == "MonthYearDate":
+            return (
+                datetime.strptime(answer_value, "%Y-%m")
+                .replace(tzinfo=timezone.utc)
+                .strftime("%m/%Y")
+            )
 
     return answer_data
 
 
-def _get_checkbox_answer_data(answer_store, answer_schema, value):
+def _get_checkbox_answer_data(
+    answer_store: AnswerStore, answer_schema: Mapping, value: AnswerValueTypes
+) -> dict:
```

Comment:
This should be more specific. `dict[str, str]` should be correct as q_code and option values are strings.

Reply:
More specific now, thanks!